barcodes_generator_package/tests/test_barcode_generator.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestBarcodeGenerator(TransactionCase):
+    def setUp(self):
```

Comment:
Maybe changing this and switching to SavepointCase

Reply:
@Raul-S73 I forgot one part of my sentence. I wanted to say changing also setUp to setUpClass :sweat_smile: